### PR TITLE
Bugfix : Responsive Error in login page 

### LINKED ISSUE #3250: Responsive Error in login page
**Login page shifts to left when width of screen decreases**
The Login div shifts to left of the screen when width of screen decreases to 767px or less, which according to me stay at centre.

**You can check your self**
Goto this url ~ https://6.demo.plone.org/login?return_url= 
reduce your screen width and see yourself.

*Screenshots*
Screenshot 2022-04-15 at 3 30 01 PM
Screenshot 2022-04-15 at 3 30 10 PM

### PR DESCRIPTION
**Pull request by suman9893**
Fixes #3250

### CODE REVIEW

**Comment by suman9893** [comment]
May I know the reason why my PR is not getting merged? So that I can improve the PR accordingly.

File: news/3250.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix for responsive error in the login page when the width of the screen decreases.
```
Comment (stevepiercy) [inline news/3250.bugfix:1]: Please add your username to the news item.
```suggestion
Fix for responsive error in the login page when the width of the screen decreases. @suman9893
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]
I approve, but it needs further review from the @plone/volto-team .

**Comment by davisagli** [comment]
@suman9893 Thank you, and sorry it took us so long to review and merge this.

**Comment by suman9893** [comment]
> @suman9893 Thank you, and sorry it took us so long to review and merge this.

May you please check #4513 ? I think this can also get merged. @davisagli

**Pull request merged**